### ISSUE TITLE
Build failed on Linux Ubuntu

### ISSUE THREAD

**Issue by jurajsomorovsky** [mitls/mitls-flex#5]
As discussed...report:

Errors:

/home/juraj/programs/mitls-flex/VS/miTLS.sln (default targets) ->
(Build target) ->
/home/juraj/programs/mitls-flex/libs/DB/DB.fsproj (default targets) ->

```
/home/juraj/programs/mitls-flex/libs/DB/DB.fsproj: error : Target named 'Build' not found in the project.
```

/home/juraj/programs/mitls-flex/VS/miTLS.sln (default targets) ->
(Build target) ->
/home/juraj/programs/mitls-flex/libs/Platform/Platform.fsproj (default targets) ->

```
/home/juraj/programs/mitls-flex/libs/Platform/Platform.fsproj: error : Target named 'Build' not found in the project.
```

Mono JIT compiler version 3.2.8 (Debian 3.2.8+dfsg-4ubuntu4)

**Comment by pfasante** [comment]
did you install all of the prerequisites?
I'm also not able to install it on xubuntu-xenial, but I get another error (which is later in the make phase, I guess):

> % make
> xbuild /nologo /p:Configuration=Release VS/miTLS.sln
> 
> Build started 7/8/2016 9:57:27 AM.
> [...]
> Errors:
> 
> /home/asante/werkstatt/werkbank/mitls-flex/VS/miTLS.sln (default targets) ->
> (Build target) ->
> /home/asante/werkstatt/werkbank/mitls-flex/libs/OpenSSL/OpenSSL.csproj (default targets) ->
> /usr/lib/mono/4.5/Microsoft.CSharp.targets (CoreCompile target) ->
> 
>   CSC: error CS0518: The predefined type `System.Object` is not defined or imported
>   [...]
>   CSC: error CS0518: The predefined type `System.Exception` is not defined or imported
> 
>    2 Warning(s)
>    31 Error(s)
> 
> Time Elapsed 00:00:07.0791400
> Makefile:26: recipe for target 'build' failed

Haven't yet found out, what I am missing, the complete make output is: [make-log.txt](https://github.com/mitls/mitls-flex/files/353770/make-log.txt).
What I've done:
- git clone git@github.com:mitls/mitls-flex.git
- sudo apt install mono-complete mono-xbuild fsharp nuget
- cd mitls-flex/VS && nuget restore && cd ..
- make

**Comment by markulf** [comment]
Is it related to differing runtime versions. Have a look at 

http://stackoverflow.com/questions/10490155/unable-to-run-net-app-with-mono-mscorlib-dll-not-found-version-mismatch
http://superuser.com/questions/743863/debian-unable-to-use-mono-missing-mscorlib-dll?

This may fix the immediate problem, but won't be a long-term solution. We should adapt the README.

**Comment by pfasante** [comment]
that's right. adding `/p:TargetFrameworkVersion="v4.5"` to the build variable in the Makefile fixed it for me (based on this so answer: https://stackoverflow.com/questions/27594393/compiled-mono-missing-default-net-libraries-system-object-is-not-defined-or-i/27855445#27855445 ).
